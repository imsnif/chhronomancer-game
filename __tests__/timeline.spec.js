import 'react-native'
import React from 'react'
import Timeline from '../components/timeline'
import renderer from 'react-test-renderer'

jest.mock('mobx-react/native', () => require('mobx-react/custom'))

test('ProgressBar (default props) => renders correctly', () => {
  const tree = renderer.create(
    <Timeline
      timelineStore={{players: []}}
      playersStore={{players: []}}
      clockStore={{time: 1}}
    />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('ProgressBar (with one player) => renders correctly', () => {
  const tree = renderer.create(
    <Timeline
      timelineStore={{players: [1]}}
      playersStore={{
        players: [
          {
            id: 1,
            name: 'foo',
            image: require('../assets/placeholders/me.jpeg'),
            items: {}
          }
        ]
      }}
      clockStore={{time: 1}}
    />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
