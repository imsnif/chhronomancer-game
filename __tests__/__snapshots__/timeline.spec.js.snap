// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar (default props) => renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#2f3241",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "borderColor": "black",
        "borderStyle": "dashed",
        "borderWidth": 5,
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      style={
        Object {
          "flex": 1,
          "padding": 5,
        }
      }
    >
      <View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
      </View>
    </RCTScrollView>
  </View>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Reset
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Quest
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Lock
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Unlock
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Travel Here
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`ProgressBar (with one player) => renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#2f3241",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "borderColor": "black",
        "borderStyle": "dashed",
        "borderWidth": 5,
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      style={
        Object {
          "flex": 1,
          "padding": 5,
        }
      }
    >
      <View>
        <View
          style={
            Object {
              "opacity": 0,
            }
          }
        >
          <View
            style={
              Object {
                "backgroundColor": "#02ffff",
                "borderColor": "#02ffff",
                "borderRadius": 2,
                "borderWidth": 0.2,
                "flex": 1,
                "flexDirection": "column",
                "height": 100,
                "margin": 5,
                "marginBottom": 15,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "row",
                }
              }
            >
              <Image
                source={1}
                style={
                  Object {
                    "backgroundColor": "#2f3241",
                    "borderRadius": 2,
                    "flex": 1,
                    "height": 21,
                    "margin": 2,
                    "marginBottom": 1,
                    "width": 10,
                  }
                }
              />
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#2e1a29",
                    "flex": 9,
                    "fontFamily": "sans-serif",
                    "paddingRight": 30,
                    "paddingTop": 2,
                    "textAlign": "center",
                  }
                }
              >
                foo
              </Text>
            </View>
            <View
              style={
                Object {
                  "backgroundColor": "#2f3241",
                  "borderColor": "#2f3241",
                  "borderRadius": 2,
                  "borderWidth": 1,
                  "flex": 3,
                  "flexDirection": "column",
                  "margin": 2,
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              />
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flex": 1,
                      "flexDirection": "row",
                      "justifyContent": "center",
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#2f3241",
                        "borderColor": "#02ffff",
                        "borderRadius": 2,
                        "borderWidth": 1,
                        "height": 20,
                        "margin": 4,
                        "width": 20,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "height": 10,
                          "width": 10,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "borderBottomColor": "red",
                            "borderBottomWidth": 2,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "transparent",
                            "borderTopWidth": 5,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                      <View
                        style={
                          Object {
                            "borderBottomColor": "transparent",
                            "borderBottomWidth": 5,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "red",
                            "borderTopWidth": 7,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                    </View>
                  </View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#2f3241",
                        "borderColor": "#02ffff",
                        "borderRadius": 2,
                        "borderWidth": 1,
                        "height": 20,
                        "margin": 4,
                        "width": 20,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "height": 10,
                          "width": 10,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "borderBottomColor": "green",
                            "borderBottomWidth": 2,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "transparent",
                            "borderTopWidth": 5,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                      <View
                        style={
                          Object {
                            "borderBottomColor": "transparent",
                            "borderBottomWidth": 5,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "green",
                            "borderTopWidth": 7,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                    </View>
                  </View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#2f3241",
                        "borderColor": "#02ffff",
                        "borderRadius": 2,
                        "borderWidth": 1,
                        "height": 20,
                        "margin": 4,
                        "width": 20,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "height": 10,
                          "width": 10,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "borderBottomColor": "blue",
                            "borderBottomWidth": 2,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "transparent",
                            "borderTopWidth": 5,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                      <View
                        style={
                          Object {
                            "borderBottomColor": "transparent",
                            "borderBottomWidth": 5,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "blue",
                            "borderTopWidth": 7,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                    </View>
                  </View>
                </View>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flex": 1,
                      "flexDirection": "row",
                      "justifyContent": "center",
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#2f3241",
                        "borderColor": "#02ffff",
                        "borderRadius": 2,
                        "borderWidth": 1,
                        "height": 20,
                        "margin": 4,
                        "width": 20,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "height": 10,
                          "width": 10,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "borderBottomColor": "purple",
                            "borderBottomWidth": 2,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "transparent",
                            "borderTopWidth": 5,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                      <View
                        style={
                          Object {
                            "borderBottomColor": "transparent",
                            "borderBottomWidth": 5,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "purple",
                            "borderTopWidth": 7,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                    </View>
                  </View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#2f3241",
                        "borderColor": "#02ffff",
                        "borderRadius": 2,
                        "borderWidth": 1,
                        "height": 20,
                        "margin": 4,
                        "width": 20,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "height": 10,
                          "width": 10,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "borderBottomColor": "brown",
                            "borderBottomWidth": 2,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "transparent",
                            "borderTopWidth": 5,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                      <View
                        style={
                          Object {
                            "borderBottomColor": "transparent",
                            "borderBottomWidth": 5,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "brown",
                            "borderTopWidth": 7,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                    </View>
                  </View>
                </View>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flex": 1,
                      "flexDirection": "row",
                      "justifyContent": "center",
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#2f3241",
                        "borderColor": "#02ffff",
                        "borderRadius": 2,
                        "borderWidth": 1,
                        "height": 20,
                        "margin": 4,
                        "width": 20,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "height": 10,
                          "width": 10,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "borderBottomColor": "black",
                            "borderBottomWidth": 2,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "transparent",
                            "borderTopWidth": 5,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                      <View
                        style={
                          Object {
                            "borderBottomColor": "transparent",
                            "borderBottomWidth": 5,
                            "borderLeftColor": "transparent",
                            "borderLeftWidth": 5,
                            "borderRightColor": "transparent",
                            "borderRightWidth": 5,
                            "borderTopColor": "black",
                            "borderTopWidth": 7,
                            "height": 0,
                            "width": 0,
                          }
                        }
                      />
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
      </View>
    </RCTScrollView>
  </View>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Reset
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Quest
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Lock
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Unlock
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "margin": 7,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "backgroundColor": "#2f3241",
              "borderColor": "#02ffff",
              "borderRadius": 2,
              "borderWidth": 2,
              "flex": 1,
              "width": "100%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02ffff",
              "paddingTop": 12,
              "textAlign": "center",
            }
          }
        >
          Travel Here
        </Text>
      </View>
    </View>
  </View>
</View>
`;
